engine/schema/src/main/resources/META-INF/db/schema-41810to41900.sql

```diff
@@ -19,3 +19,12 @@
 -- Schema upgrade from 4.18.1.0 to 4.19.0.0
 --;
 
+DROP PROCEDURE IF EXISTS `cloud`.`IDEMPOTENT_ADD_FOREIGN_KEY`;
```

Comment:
@harikrishna-patnala 
we need to fix this in 4.18.1.0, right ?

Reply:
yeah makes sense, updated it @weizhouapache